lib/widgets/fields/boolean_field.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_quest/widgets/boolean_field.dart';
+
+class BooleanField extends StatelessWidget {
```

Comment:
We are not adding or abstracting any field in this widget, then this is a useless wrapper and shouldn't exist.
Note that we wrapped AppTextField inside NumberField because NumberField is setting some values that make this field deal with numbers only. But in the current case, this enclosure is not abstracting or implementing anything.